scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -620,7 +626,7 @@ global.base_stats = { //tempory stats subject to change by anyone that wishes to
 			luck : 10,
 			technology : [8,1],
 			skills : {}, // TODO consider what skills are needed for this bloke
-			start_gear:{"armour":"Light Power Armour", "wep1":"Power Sword", "wep2":"Storm Shield"}, // TODO - add Light variant of Power Armour
+			start_gear:{"armour":"Light Power Armour", "wep1":"Power Sword", "wep2":"Combat Shield"}, // TODO - add Light variant of Power Armour
```

Comment:
Combat Shield is classified as gear item.
Are You sure this won't cause issues?